### PR TITLE
Add CDC public Nextflow pipelines

### PR DESCRIPTION
**Pull request by mribeirodantas**
Adds PHoeNIx, mycosnp-nf and SC2CLIA, three public Nextflow pipelines from CDC.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,17 +115,20 @@ Other pipelines
 * [jdidion/rna-quick](https://github.com/jdidion/rna-quick) - Rapid analysis of gene and transcript quantification and differential expression.
 * [joshua-d-campbell/nf-GATK_Exome_Preprocess](https://github.com/joshua-d-campbell/nf-GATK_Exome_Preprocess) - GATK best practice guide to preprocess whole exome sequencing (WES) data.
 * [meissnert/nextflow-gtool](https://github.com/meissnert/nextflow-gtool)
+* [mycosnp-nf](https://github.com/CDCgov/mycosnp-nf) - portable workflow for performing whole genome sequencing analysis of fungal organisms
```
Comment (pditommaso) [inline README.md]:
```suggestion
* [CDCgov/mycosnp-nf](https://github.com/CDCgov/mycosnp-nf) - Portable workflow for performing whole genome sequencing analysis of fungal organisms.
```

File: README.md [hunk 2]
```diff
@@ -115,17 +115,20 @@ Other pipelines
...
+* [CDCgov/mycosnp-nf](https://github.com/CDCgov/mycosnp-nf) - Portable workflow for performing whole genome sequencing analysis of fungal organisms.
 * [NationalGenomicsInfrastructure/icing](https://github.com/NationalGenomicsInfrastructure/icing) - HLA typing from OxfordNanopore reads.
 * [NBISweden/wgs-structvar](https://github.com/NBISweden/wgs-structvar) - Genome Sequenceing Structural Variation Pipelines.
 * [MHH-RCUG/nf_wochenende](https://github.com/MHH-RCUG/nf_wochenende) - A metagenomic (and genomic) alignment and normalization pipeline for long and short reads
 * [nmdp-bioinformatics/flow-blast-hml](https://github.com/nmdp-bioinformatics/flow-blast-hml)
 * [nmdp-bioinformatics/flow-validation](https://github.com/nmdp-bioinformatics/flow-validation)
 * [ODiogoSilva/innuca-nf](https://github.com/ODiogoSilva/innuca-nf) - A nextflow implementation of INNUENDO quality control of reads, de novo assembly and contigs quality assessment.
 * [pbelmann/sra-download](https://github.com/pbelmann/sra-download)
+* [PHoeNIx](https://github.com/CDCgov/phoenix) - PHoeNIx: A short-read pipeline for healthcare-associated and antimicrobial resistant pathogens.
```
Comment (mribeirodantas) [inline README.md]:
```suggestion
* [CDCgov/PHoeNIx](https://github.com/CDCgov/phoenix) - PHoeNIx: A short-read pipeline for healthcare-associated and antimicrobial resistant pathogens.
```

File: README.md [hunk 3]
```diff
@@ -115,17 +115,20 @@ Other pipelines
...
 * [PlantandFoodResearch/VariantAnalysis](https://github.com/PlantandFoodResearch/VariantAnalysis) - Bioinformatic variant calling pipeline.
 * [robsyme/nextflow-annotate](https://github.com/robsyme/nextflow-annotate) - Fungal genome annotation workflow
 * [robsyme/nextflow-mapping](https://github.com/robsyme/nextflow-mapping)
 * [robsyme/workflows-proteogenomics](https://github.com/robsyme/workflows-proteogenomics) - Versioned, shareable and replicable workflows for proteogenomics.
+* [SC2CLIA](https://github.com/CDCgov/SC2CLIA) - pipeline for Clinical Laboratories Improvements Amendments (CLIA) compliant variant calling and spike protein substitution prediction with additional tools for quality control
```
Comment (mribeirodantas) [inline README.md]:
```suggestion
* [CDCgov/SC2CLIA](https://github.com/CDCgov/SC2CLIA) - Pipeline for Clinical Laboratories Improvements Amendments (CLIA) compliant variant calling and spike protein substitution prediction with additional tools for quality control
```

File: README.md [hunk 4]
```diff
@@ -115,17 +115,20 @@ Other pipelines
...
+* [CDCgov/mycosnp-nf](https://github.com/CDCgov/mycosnp-nf) - Portable workflow for performing whole genome sequencing analysis of fungal organisms.
 * [NationalGenomicsInfrastructure/icing](https://github.com/NationalGenomicsInfrastructure/icing) - HLA typing from OxfordNanopore reads.
 * [NBISweden/wgs-structvar](https://github.com/NBISweden/wgs-structvar) - Genome Sequenceing Structural Variation Pipelines.
 * [MHH-RCUG/nf_wochenende](https://github.com/MHH-RCUG/nf_wochenende) - A metagenomic (and genomic) alignment and normalization pipeline for long and short reads
 * [nmdp-bioinformatics/flow-blast-hml](https://github.com/nmdp-bioinformatics/flow-blast-hml)
 * [nmdp-bioinformatics/flow-validation](https://github.com/nmdp-bioinformatics/flow-validation)
 * [ODiogoSilva/innuca-nf](https://github.com/ODiogoSilva/innuca-nf) - A nextflow implementation of INNUENDO quality control of reads, de novo assembly and contigs quality assessment.
 * [pbelmann/sra-download](https://github.com/pbelmann/sra-download)
+* [CDCgov/PHoeNIx](https://github.com/CDCgov/phoenix) - PHoeNIx: A short-read pipeline for healthcare-associated and antimicrobial resistant pathogens.
```
Comment (pditommaso) [inline README.md]:
```suggestion
* [CDCgov/PHoeNIx](https://github.com/CDCgov/phoenix) - A short-read pipeline for healthcare-associated and antimicrobial resistant pathogens.
```

**Review by pditommaso — APPROVED** [review]

**Comment by mribeirodantas** [comment]
🥳 Thanks, Paolo!

**Pull request merged**